### ISSUE TITLE
5.4.1 "the General Population"

### ISSUE THREAD

**Issue by MakotoUeki** [w3c/coga#302]
5.4.1 Differences from Usability Testing with the General Population
https://www.w3.org/TR/coga-usable/#differences-from-usability-testing-with-the-general-population

I'm not a native English speaker. However "General Population" sounds like that people with cognitive and learning disabilities are not "General Population" in this context. 

Maybe "People without Cognitive and Learning Disabilities" might be better than "General Population".

**Comment by eric-hind** [comment]
We may want to rephrase this to a positive: those who need additional considerations?

**Comment by rachaelbradley** [comment]
Maybe "Additional considerations when user testing with people with cognitive and learning disabilities." as the title. Then "People who have cognitive and learning disabilities have additional needs when participating in user testing."
